conf.yaml

```diff
@@ -1145,6 +1146,23 @@ contents:
                   -
                     repo:   docs
                     path:   shared/versions/stack/{version}.asciidoc
+              - title:      Enterprise Search Node.js client (beta)
+                prefix:     enterprise-search-js
+                private:    1
+                single:     1
+                current:    *stackcurrent
+                branches:   [ {main: master}, 8.3, 8.2, 8.1, 8.0, 7.17 ]
```

Comment:
I don't think there are branches for any version besides `main` for the client. Probably should update this value?